contrast/aws-integration.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Minor nit: this would be more portable.

```suggestion
#!/usr/bin/env bash
```